build-tools/packages/build-cli/src/commands/bump/version.ts

```diff
@@ -0,0 +1,199 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import fetch from "node-fetch";
+import { promises as fsPromises } from "fs";
+import { Flags } from "@oclif/core";
+import { Logger } from "@fluidframework/build-tools";
+import { BaseCommand } from "../../base";
+
+const PACKAGE_NAME = "@fluidframework/container-runtime";
+const REGISTRY_URL =
+	"https://pkgs.dev.azure.com/fluidframework/internal/_packaging/build/npm/registry/";
+const organization = "fluidframework";
+const project = "internal";
+const ADO_BASE_URL = `https://dev.azure.com/${organization}/${project}/_apis/build/builds?api-version=7.0`;
+const GITHUB_RELEASE_URL = "https://api.github.com/repos/microsoft/FluidFramework/releases";
+
+interface Build {
```

Comment:
nit: more descriptive interface name
```suggestion
interface IBuildDetails {
```